AmplifyPlugins/Storage/AWSS3StoragePlugin/Dependency/AWSS3PreSignedURL+PutObjectInput.swift

```diff
@@ -1,52 +0,0 @@
-//
-// Copyright Amazon.com Inc. or its affiliates.
-// All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-
-import Foundation
-
-import AWSS3
-import ClientRuntime
-import AWSClientRuntime
-
-extension PutObjectInput {
```

Comment:
Why are these removed? Not used?

Reply:
Correct.  These were a workaround for some bugs in the previous version of the SDK.  Most of these issues have been fixed, so they are no longer needed.  There's still one issue that still hasn't been fixed in the current version of the SDK, so I added an updated version of one of the workarounds (I updated it to work properly with the async version) back in.